README.md

```diff
@@ -13,7 +13,7 @@ The Chartboost Mediation Digital Turbine Exchange adapter mediates Digital Turbi
 
 In your `build.gradle`, add the following entry:
 ```
-    implementation "com.chartboost:chartboost-mediation-adapter-digital-turbine-exchange:4.8.2.3.0"
+    implementation "com.chartboost:chartboost-mediation-adapter-digital-turbine-exchange:4.8.2.1.1"
```

Comment:
4.8.2.2.1?

Reply:
Copy/Paste, but forgot to edit 🤦

Reply:
4.8.2.2.1 or 4.8.2.1.1?

Reply:
Jesus. How did I mess that up again

Reply:
All good. 😄

Reply:
Wait.. wth. It shows 4.8.2.2.1 locally

Reply:
I guess I hadn't pushed my change 🤨 SourceTree showed everything as pushed though. Weird